system-tests/project-fixtures/react/webpack.config.js

```diff
@@ -18,6 +18,11 @@ module.exports = {
         exclude: /node_modules/,
         use: {
           loader: 'babel-loader',
+          options: {
```

Comment:
I had to add these, I'm not sure why it was not picking up the `.babelrc`. Either way, it's unrelated to this bug fix - although there might be another, unrelated issue around babel-loader not automatically finding `.babelrc` (that was not introduced by this PR).

If that is an issue, can look into fixing it separately - no bandwidth to go deeper into specifics right now.

Reply:
Was this working before the changes in this PR?   How did you know that babel was not loading correctly?  Did you see an error?

Reply:
It is complaining about not knowing about JSX - I think what's happening is some other part of the toolchain (webpack) expects `.babelrc` to exist in the same directory where the process is running, which is now a **nested** directory (in `cypress/` in this case.

I am not sure if this going to impact users. It's likely this is a problem no-one has encountered, since you can't current run with a nested config file, anyway.

Either way, I will create a more minimal project are requested that will side step this issue. Projects using babel and relying on `.babelrc` might run into this, but it'll be a separate and unrelated bug...

Reply:
I pushed a simple project that won't deal with this. The fact webpack relies on the cwd to find the `babelrc` might be an issue but I'm not sure that's even our problem to solve.